### DISCUSSION TITLE
Inline code overlap

### DISCUSSION THREAD

**Discussion by fabriziobagala** [standardnotes/forum#2775] [General]
### Steps to reproduce

1. Open the web app
2. Select the "Markdown Pro" editor
3. Create a new note
4. Add a numbered list
5. Create multiple lines in the numbered list by inserting inline code for each line

### Actual result

The inline code of the second line overlaps the inline code of the first line.

### Expected result

The two inline codes do not overlap.

### Screenshot

![immagine](https://user-images.githubusercontent.com/62190484/147641412-47e5930d-a972-4315-9e5a-a9d226b749a0.png)

### Web app version

3.9.10

**Comment by BobWs** [comment]
This problem is still present after 3 years. Guess it is a low priority….
